rllib/optimizers/sync_replay_optimizer.py

```diff
@@ -51,25 +54,40 @@ def __init__(self,
             prioritized_replay_alpha (float): replay alpha hyperparameter
             prioritized_replay_beta (float): replay beta hyperparameter
             prioritized_replay_eps (float): replay eps hyperparameter
-            schedule_max_timesteps (int): number of timesteps in the schedule
-            beta_annealing_fraction (float): fraction of schedule to anneal
-                beta over
-            final_prioritized_replay_beta (float): final value of beta
+            schedule_max_timesteps (int): DEPRECATED: number of timesteps in
+                the schedule to anneal PR-beta.
+            beta_annealing_fraction (float): DEPRECATED: Fraction of schedule
+                to anneal PR-beta over.
+            final_prioritized_replay_beta (float): Final value of beta.
             train_batch_size (int): size of batches to learn on
-            sample_batch_size (int): size of batches to sample from workers
             before_learn_on_batch (function): callback to run before passing
                 the sampled batch to learn on
             synchronize_sampling (bool): whether to sample the experiences for
                 all policies with the same indices (used in MADDPG).
+            prioritized_replay_beta_annealing_timesteps (int): The timestep at
+                which PR-beta annealing should end.
         """
         PolicyOptimizer.__init__(self, workers)
 
         self.replay_starts = learning_starts
+
+        # Deprecated parameters.
+        if prioritized_replay_beta_annealing_timesteps is None:
+            assert schedule_max_timesteps and beta_annealing_fraction
+            prioritized_replay_beta_annealing_timesteps = \
+                schedule_max_timesteps * beta_annealing_fraction
+        if schedule_max_timesteps is not None:
+            deprecation_warning("schedule_max_timesteps",
+                                "prioritized_replay_beta_annealing_timesteps")
+        if beta_annealing_fraction is not None:
+            deprecation_warning("beta_annealing_fraction",
+                                "prioritized_replay_beta_annealing_timesteps")
```

Comment:
No need to check deprecation in DeveloperAPI.

Reply:
Removed all checks in sync_replay_opt.